src/components/dialogs/create-study-dialog.js

```diff
@@ -365,8 +422,50 @@ export const CreateStudyDialog = ({ open, onClose }) => {
                         style={{ width: '90%' }}
                         label={<FormattedMessage id="studyDescription" />}
                     />
-                    {caseExist && <SelectCase />}
-                    {!caseExist && <UploadCase />}
+                    {!providedCase ? (
+                        (caseExist && <SelectCase />) ||
+                        (!caseExist && <UploadCase />)
+                    ) : (
+                        <>
+                            <table>
```

Comment:
Why using a table here ?

Reply:
I wanted to reproduce the same display as in the UploadCase component, so for the sake of homogeneity I inserted the same HTML structure.
I'll see how to simplify this, should I update UploadCase too?

Reply:
If it's possible, maybe you can reuse the UploadCase component and change this component to use the material iu Table component ?

Reply:
I don't think it will do the trick since i was inspired by UploadCase only for its layout, its logic is not what i'm looking for here.
For now I've recreated the same view by replacing `<table>` simply by a `<div>` and adding a few margins.